CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+## Contributing
+
+All pull requests should be made to the develop branch.
+
+```
+git clone https://github.com/vinnymac/PokeNurse
+cd PokeNurse
+git checkout develop
+yarn
+yarn run dev
+```
+
+## Login related issues
+
+Note that issues regarding login are better handled in [pogobuf](https://github.com/pogosandbox/pogobuf) as this project is reliant on it for access.
```

Comment:
Change the URL to https://github.com/cyraxx/pogobuf, since that is the one we use.